x-pack/plugins/synthetics/server/lib/synthetics_service/normalizers/browser.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DEFAULT_FIELDS } from '../../../../common/constants/monitor_defaults';
+
+import {
+  BrowserFields,
+  ConfigKey,
+  DataStream,
+  Locations,
+  PushBrowserMonitor,
+  ScheduleUnit,
+} from '../../../../common/runtime_types/monitor_management';
+
+/* Represents all of the push-monitor related fields that need to be
+ * normalized. Excludes fields that we do not support for push monitors
+ * This type ensures that contributors remember to add normalizers for push
+ * monitors where appropriate when new keys are added to browser montiors */
+type NormalizedPublicFields = Omit<
+  BrowserFields,
+  | ConfigKey.METADATA
+  | ConfigKey.SOURCE_INLINE
+  | ConfigKey.SOURCE_ZIP_URL
+  | ConfigKey.SOURCE_ZIP_USERNAME
+  | ConfigKey.SOURCE_ZIP_PASSWORD
+  | ConfigKey.SOURCE_ZIP_FOLDER
+  | ConfigKey.SOURCE_ZIP_PROXY_URL
+  | ConfigKey.ZIP_URL_TLS_CERTIFICATE_AUTHORITIES
+  | ConfigKey.ZIP_URL_TLS_CERTIFICATE
+  | ConfigKey.ZIP_URL_TLS_KEY
+  | ConfigKey.ZIP_URL_TLS_KEY_PASSPHRASE
+  | ConfigKey.ZIP_URL_TLS_VERIFICATION_MODE
+  | ConfigKey.ZIP_URL_TLS_VERSION
+  | ConfigKey.JOURNEY_FILTERS_TAGS
+  | ConfigKey.SYNTHETICS_ARGS
+  | ConfigKey.PORT
+  | ConfigKey.URLS
+  | ConfigKey.ENABLED // should we allow enabled?
+  | ConfigKey.NAMESPACE // should we allow namespace?
+>;
+
+export const normalizePushedMonitor = ({
+  locations = [],
+  monitor,
+  projectId,
+}: {
+  locations: Locations;
+  monitor: PushBrowserMonitor;
+  projectId: string;
+}): BrowserFields => {
+  const defaultFields = DEFAULT_FIELDS[DataStream.BROWSER];
+  const normalizedFields: NormalizedPublicFields = {
+    [ConfigKey.MONITOR_TYPE]: DataStream.BROWSER,
+    [ConfigKey.IS_PUSH_MONITOR]: true,
+    [ConfigKey.NAME]: monitor.name || '',
+    [ConfigKey.SCHEDULE]: {
+      number: `${monitor.schedule}`,
+      unit: ScheduleUnit.MINUTES,
+    },
+    [ConfigKey.PROJECT_ID]: projectId || defaultFields[ConfigKey.PROJECT_ID],
+    [ConfigKey.JOURNEY_ID]: monitor.id || defaultFields[ConfigKey.JOURNEY_ID],
```

Comment:
i think it's confusing that it's we are mapping journey id to monitor id. Why not keep it as monitor id everywhere?